db/migrate/20201215200431_change_authenticable_id_to_user_id.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAuthenticableIdToUserId < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :authentication_tokens, :authenticable_id, :user_id
```

Comment:
@davidgisbey We may get some issues (inserting into non existent column) if users are authenticating post migration but before before the rest of the logic is live?

Reply:
Yea i'll merge and deploy out the migration separately once we're happy that's the way to go. Good shout.